package exeption.dz2;

public class Task2 {
    public static void main(String[] args) throws Exception {
    
        // System.out.println("hello world");
        // float res = FuncTask2.getFloat();
        // System.out.println(res);
        //FuncTask2.execercise1();
        FuncTask2.exercize2();
    }     
    
    
}
